app/presenters/linkable.rb

```diff
@@ -1,10 +1,17 @@
 module Linkable
   def from
-    links("lead_organisations") + links("supporting_organisations") + links("worldwide_organisations")
+    links("lead_organisations") +
```

Comment:
You'll need to run all this through `uniq`, as some orgs or policies may appear in more than one of these. Same for `part_of`